apps/inviwo/CMakeLists.txt

```diff
@@ -173,3 +177,5 @@ endif()
 # Save information for python tools.
 ivw_mod_name_to_class(ivw_module_classes ${enabled_modules})
 ivw_private_create_pyconfig("${IVW_MODULE_DIR};${IVW_EXTERNAL_MODULES}" "${ivw_module_classes}" inviwo)
+
+include(packaging/packaging.cmake)           # Package creation
```

Comment:
What is the semantics in CMake for running CPack from a subdir and not at the top level in the project? 
Is it even possible to run cpack more then ones? how would that even work?

Reply:
Perhaps using CPACK_INSTALL_CMAKE_PROJECTS would be an additional way. Slicer seem to use it heavily and includes cpack last
https://github.com/Slicer/Slicer/blob/master/CMake/LastConfigureStep/CMakeLists.txt

Reply:
They use a "main project", which determines which app to pack

Reply:
hehe, they have a CMake setup that is possibly even more complicated then ours :)
Yes, that might be an option to explore...

Reply:
Still, since this works, I think we can keep it. If we discover issues we can consider changing. I think that this way is a bit cleaner than using a "main project" because one can more easily customize the packaging